### PR TITLE
Create Seed script

### PR DESCRIPTION
**Pull request by jksf**
Compiles contracts, starts TestRPC and runs migrations.

It breaks webapp because there is no way to run seed without compiling typescript first. Also `testrpc:deploy` is changed to `seed`.

I will publish compiled js to public npm repositories and avoid versioning of built js files.

### CODE REVIEW

File: types/web3.d.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
+type Callback<T> = (err: Error | null, value: T) => void;
+type VoidCallback = (err: Error | null) => void;
+
+type Address = string;
+
+declare module 'web3' {
+  import { BigNumber } from 'bignumber.js';
+
+  type AnyNumber = number | string | BigNumber;
```
Comment (biern) [inline types/web3.d.ts:9]: `NumberLike` in `bignumber.js` is exaclty that

**Review by biern — COMMENTED** [review]
Looks good, the webapp doesn't rely on a seed functionality. 
However, I'm not comfortable with merging it until the #33 has the ability to run migrations as a standalone script. Otherwise, master will be unusable by the current webapp master.

File: types/web3.d.ts (hunk 1 above)
Comment (biern) [reply types/web3.d.ts:9]: I know that #33 depends on this PR, so don't bother fixing it in this branch, it should save you some headaches.

Comment (jksf) [reply types/web3.d.ts:9]: `NumberLike` is defined in `@types/bignumber.js` which is not compatible with the newest `bignumber.js` package. The newest `bignumber.js` package ships with its own typings, unfortunately without `NumberLike` type.

**Comment by jksf** [comment]
I fixed the issues related to compatibility with the webapp. 
PR #33 is not dependent on this PR. It's the opposite. Feel free to merge it if it complies with the webapp.

File: types/web3.d.ts (hunk 1 above)
Comment (jksf) [reply types/web3.d.ts:9]: https://github.com/MikeMcl/bignumber.js/blob/master/bignumber.d.ts

Comment (biern) [reply types/web3.d.ts:9]: weird, good to know that

**Pull request merged**